### ISSUE TITLE
Processing SplitSeq / ParseBio data with simpleaf

### ISSUE THREAD

**Issue by wmacnair** [COMBINE-lab/simpleaf#104]
Hi @rob-p and team

We have some ParseBio data that it would be nice to process with `simpleaf`.

I found this tutorial for `alevin-fry`:
https://combine-lab.github.io/alevin-fry-tutorials/2022/split-seq/
It looks like the only difference to what `simpleaf` wraps around is using a `--splitseqV1` flag in the `salmon alevin` call. However I searched the `simpleaf` documentation page for "splitseq" / "split seq", and that didn't turn up any results. 

Does that mean that at the moment, `simpleaf` isn't set up for splitSeq / ParseBio fastq formats? If that's the case, please add a small +1 in favour of this task on your list of possible future tasks :)

Thanks!
Will

**Comment by rob-p** [comment]
Thanks @wmacnair!  I think this would be a prime candiate for either our upcoming seqproc feature (cc @noahcape & @Daniel-Liu-c0deb0t) or our the workflow + patch system.  Right now, one could do this with simpleaf, but they would have to process the reads with splitp first (though it supports streaming transformation).

--Rob

**Comment by wmacnair** [comment]
Ah ok, so a very quick fix would be updating the splitseq tutorial to show how to do it with `simpleaf`?

Will

**Comment by rob-p** [comment]
Yes, I believe so. I'll chat with Dongze about this and ping you when we have something.

Best,
Rob
